apps/full-site-editing/full-site-editing-plugin/full-site-editing/plugins/close-button-override/index.js

```diff
@@ -60,27 +82,24 @@ domReady( () => {
 			thinContent.innerHTML = abbreviatedContent;
 			thinContent.className = 'close-button-override-thin';
 			newCloseButton.prepend( thinContent );
-		} else if ( 'page' === editorPostType ) {
-			newCloseButton.href = 'edit.php?post_type=page';
-			newLabel = __( 'Pages' );
-		} else if ( 'post' === editorPostType ) {
-			newCloseButton.href = 'edit.php?post_type=post';
-			newLabel = __( 'Posts' );
-			newCloseButton.setAttribute( 'aria-label', newLabel );
+			componentsToolbar.prepend( newCloseButton );
 		}
 
+		// Create a normal back button for the page/post editor.
```

Comment:
Was there a reason why we decided to go with a custom button for the template part's "Back to page"? 🤔

Reply:
It was a design decision, we did previously try to change this to match the back button on the normal editors screens.  Hopefully I am conveying the reasoning properly, but since the template part editor leads back to another instance of the editor (the page editor) it shouldn't have the same icon that represents closing or leaving the editor all together.